pom.xml

```diff
@@ -1006,6 +1008,33 @@
         <artifactId>nuxeo-platform-pdf-utils</artifactId>
         <version>${nuxeo.pdfutils.version}</version>
       </dependency>
+
+      <!-- 3D -->
+      <dependency>
+        <groupId>org.nuxeo.ecm.platform</groupId>
+        <artifactId>nuxeo-platform-3d-api</artifactId>
+        <version>${nuxeo.threed.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.nuxeo.ecm.platform</groupId>
+        <artifactId>nuxeo-platform-3d-core</artifactId>
+        <version>${nuxeo.threed.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.nuxeo.ecm.platform</groupId>
+        <artifactId>nuxeo-platform-3d-convert</artifactId>
```

Comment:
We went overboard in the past with splitting things in many bundles. We'll do a refactor pass to merge many of those in the future.